server/jetstream_cluster.go

```diff
@@ -46,6 +46,12 @@ type jetStreamCluster struct {
 	consumerResults *subscription
 }
 
+// Used to guide placement of streams in clustered JetStream.
+type Placement struct {
+	Cluster string   `json:"cluster"`
+	Tags    []string `json:"tags,omitempty"`
```

Comment:
cluster not having omitempty, does this imply that cluster is required? seems empty is valid so just checking

Reply:
We could make optional, for now it is required.

Reply:
hmm, appears to me we do handle it being empty, which looks like it then does random assignment? That would be what I'd expect anyway

Reply:
Yes if its empty we pick point of origin. What I meant was if you are filling this in you should have cluster for now.. But I can make optional or you can submit quick PR for omitempty addition.

Reply:
can wait till we do tag matching and all :)